crates/ruma-client-api/src/r0/session/login.rs

```diff
@@ -152,6 +152,8 @@ pub struct IdentityServerInfo {
 
 mod user_serde;
 
+pub mod fallback;
```

Comment:
This should be a sibling module to `login`, not a submodule. Our endpoint definitions are currently all at the same nesting depth of the module tree and I'd like to keep that. Could be named `get_login_fallback_page` or sth. like that.

Reply:
I named it `[...]::session::fallback` for now